test/test_data/position/test_nmea_petrzalka.txt

```diff
@@ -0,0 +1,13 @@
+﻿$GPGGA,135115,4806.1849,N,01706.3857,E,1,00,,171.300,M,,M,,*73
+$GPGSV,2,1,8,12,81,349,00,24,53,153,00,25,48,277,00,32,31,297,00*47
+$GPGSV,2,2,8,02,30,122,28,06,29,070,26,29,19,209,00,19,19,045,00*4C
+$GLGSV,3,1,9,78,60,325,00,87,58,087,00,77,49,087,00,88,40,157,00*5B
+$GLGSV,3,2,9,69,24,311,00,68,19,258,00,86,14,024,00,79,12,297,00*52
+$GLGSV,3,3,9,76,03,106,00,,,,,,,,,,,,*69
+$GBGSV,4,1,14,437,78,306,00,428,51,156,00,423,45,113,00,411,25,043,00*6A
+$GBGSV,4,2,14,408,22,081,00,413,21,092,00,405,21,130,00,410,17,073,00*64
+$GBGSV,4,3,14,419,11,303,00,427,10,194,00,420,07,256,00,402,06,108,00*60
+$GBGSV,4,4,14,434,06,035,00,422,01,350,00,,,,,,,,*6E
+$HCHDG,009.2,,,005.0,E*27
+$HCHDT,014.2,T*2E
+$GPMDA,,I,,B,,C,,C,,,,C,,T,,M,,N,,M*0D\r\n$IIMMB,,I,,B*49
```

Comment:
intentionally added?

Reply:
Yeah, this was more of a accidental push .. I had an nmea string that I want to use in autotests. It is not used currently but will be in autotests. I guess we can keep it here?

Reply:
sure we can keep it for later...